NiBAx/plugins/harmonization/harmonization.py

```diff
@@ -345,6 +347,43 @@ def OnAddToDataFrame(self):
         self.datamodel.data.loc[:,RAW_Residuals] = self.MUSE[RAW_Residuals]
         self.datamodel.data_changed.emit()
 
+
+    def OnMUSEharmonizationModelChanged(self):
+        if self.datamodel.IsValidHarmonization():
+            self.ui.Harmonization_Model_Loaded_Lbl.setHidden(False)
+            self.ui.Harmonization_Model_Loaded_Lbl.setObjectName('correct_label')
```

Comment:
We don't need to set the objectName again. It is being set earlier. Also, we might not even need to set the object name, we might be able to set the stylesheet without it too. What happens if you don't provide objectName in the setstylesheet call?

Reply:
Not sure what happens if the object name is not set.